droidlet/dashboard/web/src/StateManager.js

```diff
@@ -194,6 +198,20 @@ class StateManager {
     return localStorage.getItem("turk_worker_id");
   }
 
+  enableFrontendValidation() {
+    this.frontend_validation_enabled = true;
+  }
+
+  isFrontendValidationEnabled() {
+    if (
+      this.frontend_validation_enabled &&
+      this.frontend_validation_enabled == true
```

Comment:
can there be other values for `this.frontend_validation_enabled` if not aren't these two the same ?